util/configuration.py

```diff
@@ -130,8 +130,8 @@ class Email:
     def build(cls):
         email_address = ci.email_input("What is the gmail address you would " \
             "like me to use to send you reports?", provider="gmail")
-        password = ci.confirm_input("What is the password to the account you " \
-            "just enetered? ")
+        password = ci.password_input("What is the password to the account you" \
+            " just enetered?")
```

Comment:
Seeing as you only updated the method name, it would seem as both of us missed the glaring typo: "enetered". It should probably be corrected in this PR.

Reply:
Fixed!